code/game/objects/items/devices/flashlight.dm

```diff
@@ -104,40 +104,58 @@
 			to_chat(user, "[span_warning("\The [src] isn't bright enough to see anything!")] ")
 			return
 
+		var/render_list = list()//information will be packaged in a list for clean display to the user
+
 		switch(user.zone_selected)
 			if(BODY_ZONE_PRECISE_EYES)
 				if((M.head && M.head.flags_cover & HEADCOVERSEYES) || (M.wear_mask && M.wear_mask.flags_cover & MASKCOVERSEYES) || (M.glasses && M.glasses.flags_cover & GLASSESCOVERSEYES))
 					to_chat(user, span_warning("You're going to need to remove that [(M.head && M.head.flags_cover & HEADCOVERSEYES) ? "helmet" : (M.wear_mask && M.wear_mask.flags_cover & MASKCOVERSEYES) ? "mask": "glasses"] first!"))
 					return
 
 				var/obj/item/organ/internal/eyes/E = M.get_organ_slot(ORGAN_SLOT_EYES)
+				var/obj/item/organ/internal/brain = M.get_organ_slot(ORGAN_SLOT_BRAIN)
 				if(!E)
 					to_chat(user, span_warning("[M] doesn't have any eyes!"))
 					return
 
+				if(user.combat_mode)
+					M.flash_act(visual = TRUE)//Channel your anger into shining the light in their eyes for the defualt duration (~2.5s)
+				else
+					M.flash_act(visual = TRUE, length = 1 SECONDS)//breifly apply flash affect to target(~1s)
+
 				if(M == user) //they're using it on themselves
```

Comment:
You've had two other maintainers pass over this _without_ requiring this so I would feel bad to do it at this late stage but it'd be great if someone replaced these single letter vars with descriptive names.

Reply:
I would do. But 'M' seems to be a pretty standard variable used in attack calls like this. (The same variable is used in the calls governing stethoscopes too). So I think consistency would win out over descriptiveness in this case. There's loads of comments around, so hopefully it's not too obtuse to read.

Reply:
It's only consistent because all of that code is old too, but I'm not making you do it 😅